Documentation/DocuBlocks/Rest/Collections/post_api_collection.md

```diff
@@ -31,7 +31,7 @@ collections in very special occasions, but normally a regular collection will do
 @RESTBODYPARAM{schema,object,optional,}
 Optional object that specifies the collection level schema for
 documents. The attribute keys `rule`, `level` and `message` must follow the
-rules documented in [Document Schema Validation](https://www.arangodb.com/docs/devel/document-schema-validation.html)
+rules documented in [Document Schema Validation](https://www.arangodb.com/docs/stable/document-schema-validation.html)
```

Comment:
Why does it have `stable` rather then `3.8`?

Reply:
The feature has not changed, so any version from 3.7 on is fine, but `/stable` is the preferred one as it won't become stale. We used to have some links hardcoded to specific versions and they were never updated so that users ended up on obsolete documentation versions. Ideally, we would avoid absolute links altogether but relative links are supported in Swagger UI.